configure.ac

```diff
@@ -42,18 +46,27 @@ if [[ "${build_freebsd}" = "yes" ]]; then
 	LIBS="-lgeom ${LIBS}"
 fi
 
+if [[ "${build_macos}" = "yes" ]]; then
+	LIBS="-lzpool ${LIBS}"
+fi
+
 
 dnl We allow setting a zof variable which the users can set to force autoconf to consider the system is zof based
 
 AC_ARG_VAR(zof, AS_HELP_STRING([Force configure to run ZoL(f) checks]))
 AC_ARG_VAR(freebsd_src, AS_HELP_STRING([Provide path for freebsd source tree. This defaults to /usr/src/]))
+AC_ARG_WITH([o3x], AS_HELP_STRING([--with-o3x=PATH], [path to ZFS on OSX source tree. Defaults to ../zfs/]))
 
 if [[ "$prefix" = "NONE" ]]; then
 	header_prefix=/usr/local
 else
 	header_prefix=$prefix
 fi
 
+if [[ "$build_macos" = "yes" ]] && [[ "x$with_o3x" = "x" ]]; then
+	with_o3x="../zfs/"
```

Comment:
Do you think `with_o3x` is necessary ?
If it's just the path to header files, why can't we use `header_prefix` ?

Reply:
Standard o3x installation does not include headers. You need to pull the source to get the headers. Hence with_o3x allows one to specify the location of o3x source tree.